example-new-architecture/ios/File.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  File.swift
+//  DojoPaySdkExample
+//
+//  Created by Lewis Yearsley on 20/11/2024.
+//
+
+import Foundation
+import dojo_ios_sdk
```

Comment:
Was this still needed? The line and the file itself?

Reply:
Removed